deployment/sklearn/sklearn-census-end-to-end.ipynb

```diff
@@ -80,9 +76,10 @@
     "# os.environ['VERTA_HOST'] = \n",
     "\n",
     "from verta import Client\n",
+    "\n",
     "PROJECT_NAME = \"Census\"\n",
     "EXPERIMENT_NAME = \"sklearn\"\n",
-    "client = Client(os.environ['VERTA_HOST'])\n",
+    "client = Client()\n",
```

Comment:
Manually passing in host is unnecessary—we should be demonstrating that it can be picked up from the environment.